util/chplenv/__init__.py

```diff
@@ -19,6 +19,10 @@
     'chpl_timers',
     'chpl_unwind',
     'chpl_wide_pointers',
+    # et cetera
+    'chpl_home_utils',
+    'compiler_utils',
+    'overrides',
```

Comment:
Will alphabetize within this section, but keeping these util modules separate from others, similar to the third party helpers.